services/wallet/router/routes/router_path.go

```diff
@@ -54,3 +54,118 @@ type Path struct {
 func (p *Path) Equal(o *Path) bool {
 	return p.FromChain.ChainID == o.FromChain.ChainID && p.ToChain.ChainID == o.ToChain.ChainID
 }
+
+func (p *Path) Copy() *Path {
```

Comment:
I think it would be good to unit-test such function

Reply:
Added, thanks for reminding me. ;)